internal/cloud_connector/message_handlers.go

```diff
@@ -260,8 +259,8 @@ func handleOfflineMessage(client MQTT.Client, clientID domain.ClientID, msg prot
 	return nil
 }
 
-func handleEventMessage(client MQTT.Client, clientID domain.ClientID, msg protocol.ControlMessage) error {
-	logger := logger.Log.WithFields(logrus.Fields{"client_id": clientID})
+func handleEventMessage(logger *logrus.Entry, client MQTT.Client, clientID domain.ClientID, msg protocol.ControlMessage) error {
+	logger = logger.WithFields(logrus.Fields{"client_id": clientID})
```

Comment:
This line can be removed now as we are setting the client_id in the logger before this function is called.